apps/src/templates/page.html.ejs

```diff
@@ -78,6 +78,7 @@
 </div>
 
 <div id="visualizationResizeBar">
+  <i id="visualizationResizeIcon" class="fa fa-ellipsis-v"></i>
```

Comment:
here is the particular icon in question: http://fortawesome.github.io/Font-Awesome/icon/ellipsis-v/